openshift_metrics/openshift_prometheus_metrics.py

```diff
@@ -0,0 +1,62 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import argparse
+import datetime
+import os
+import sys
+
+import openshift as oc
```

Comment:
I'm try to avoid unnecessary abbreviations (because that way it's more clear what's happening in the code if you don't have the `import` section right in front of you). You only call the `openshift` module once here; I'm not sure that there's any advantage to abbreviating it to `oc`. This is also true in `utils.py`, where again there's only a single call to a module method.

Reply:
Ah, fair enough! Fixed.